[ActionList.Item] Moves CSS variables for `ActionList.Item` (#6298)
Shopify/polaris: .changeset/nine-files-reflect.md

---
'@shopify/polaris': patch
---

> [ActionList.Item] Moves CSS variables `ActionList.Item` so that it can be used standalone

Reviewer: ```suggestion
Moved item-related CSS variables to the `ActionList` `.Item` class
```
